src/bundle/ui-dev/src/modules/multi-file-upload/components/drop-area/drop.area.component.js

```diff
@@ -122,7 +131,7 @@ export default class DropAreaComponent extends Component {
         });
 
         return (
-            <form className="c-drop-area" multiple={true} onDrop={this.handleUpload}>
+            <form className="c-drop-area" onDrop={this.handleUpload} ref={this._refForm}>
```

Comment:
We need to use standard `addEventListener` because of how React handles the events, namely React has one event listener on the root instead of individual elements and an event listener from `MultiFileUploadModule` added with `removeEventListener` was called before `onDrop` listener even though onDrop is attached on a deeper node in the DOM.